### PR TITLE
WaitFor TextPresent

### PR DESCRIPTION
**Pull request by msaperst**
Adding new waitFor text present/not to align with asserts
Changing location to url for alignment (breaking change)
Editing version to match SemVer

Addresses #141

### CODE REVIEW

File: src/main/java/com/coveros/selenified/application/WaitFor.java [hunk 1]
```diff
@@ -216,19 +238,65 @@ public void location(double seconds, String location) {
...
+     * Wait up to a specified time for text to not be present on the page
+     *
+     * @param seconds - the number of seconds to wait
+     * @param text    - the to wait to not be present
+     */
+    public void textNotPresent(double seconds, String text) {
+        String action = UPTO + seconds + " seconds for page to not contain text <i>" + text + "</i>";
+        String expected = "Page doesn't contain text <b>" + text + "</b>";
+        double end = System.currentTimeMillis() + (seconds * 1000);
+        while (file.getApp().is().textPresent(text) && System.currentTimeMillis() < end) ;
```
Comment (grasbergerm) [inline src/main/java/com/coveros/selenified/application/WaitFor.java]: should we have a reference to app in the class like the other classes in this package?

Comment (msaperst) [reply src/main/java/com/coveros/selenified/application/WaitFor.java]: What do you mean?

Comment (grasbergerm) [reply src/main/java/com/coveros/selenified/application/WaitFor.java]: Like Assert.java has an app, 
`public Assert(App app, OutputFile file) {
    this.app = app;
    this.file = file;
}`
Just for consistencies sake, and cleaner code.

Comment (msaperst) [reply src/main/java/com/coveros/selenified/application/WaitFor.java]: gotcha, makes sense

**Comment by coveros-ci-dev** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [WaitFor.java#L26](https://github.com/Coveros/selenified/blob/88499b4817b5b99cf8beb6a93a57fbf4e08e2726/src/main/java/com/coveros/selenified/application/WaitFor.java#L26): Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by coveros-ci-dev** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [WaitFor.java#L26](https://github.com/Coveros/selenified/blob/988fac894a44c90fb60926e21b8344418a052fe5/src/main/java/com/coveros/selenified/application/WaitFor.java#L26): Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**